metadata/src/lib.rs

```diff
@@ -704,10 +704,6 @@ impl<'a> CustomValueMetadata<'a> {
     }
 
     /// Calculates the hash for the CustomValueMetadata.
-    ///
-    /// # Panics
-    ///
-    /// Panics if `self.type_id` is not registered in the provided type registry
```

Comment:
Is this not true anymore?

Can't find anything in this PR about this :P

Reply:
I think the `pub fn get_type_hash` would still panic if the `id` passed as param is not present in the `PortableRegistry` 🤔 

I belive the `CustomValueMetadata` could only panic in practice here if the metadata is broken, since that's where our value is coming from

Reply:
The comment was left over from when this call was taking in a `PortableRegistry` as an argument (which might then have different types from what he metadata knew about and thus could panic). That was since removed but I noticed that the comment wasn't updated so I tweaked it :)